test/sys/test_socket.rs

```diff
@@ -1535,3 +1535,114 @@ pub fn test_vsock() {
     close(s1).unwrap();
     thr.join().unwrap();
 }
+
+// Disable the test on emulated platforms because it fails in Cirrus-CI.  Lack of QEMU
+// support is suspected.
+#[cfg_attr(any(not(any(target_arch = "x86_64")), any(target_env = "musl")), ignore)]
```

Comment:
This can't be right.  CI doesn't use QEMU for x86-64-linux-musl.  What error do you get on that platform?

Reply:
Fixed the leftover from before the libc upgrade.